cli/datapipelines.go

```diff
@@ -0,0 +1,309 @@
+		Id: args.ID,
+	})
+	if err != nil {
+		return fmt.Errorf("error getting data pipeline: %w", err)
+	}
+	pipeline := resp.GetDataPipeline()
+
+	runsResp, err := client.datapipelinesClient.ListPipelineRuns(context.Background(), &datapipelinespb.ListPipelineRunsRequest{
+		Id:       args.ID,
+		PageSize: 1,
+	})
+	if err != nil {
+		return fmt.Errorf("error listing pipeline runs: %w", err)
+	}
+	runs := runsResp.Runs
+
+	mql, err := mqlJSON(pipeline.GetMqlBinary())
+	if err != nil {
+		warningf(c.App.Writer, "error parsing MQL query: %s", err)
+		mql = "(error parsing MQL query)"
+	}
+
+	printf(c.App.Writer, "ID: %s", pipeline.GetId())
+	printf(c.App.Writer, "Name: %s", pipeline.GetName())
+	printf(c.App.Writer, "Enabled: %t", pipeline.GetEnabled())
+	printf(c.App.Writer, "Schedule: %s", pipeline.GetSchedule())
+	printf(c.App.Writer, "MQL query: %s", mql)
+
+	var pipelineRunStatusMap = map[datapipelinespb.PipelineRunStatus]string{
+		datapipelinespb.PipelineRunStatus_PIPELINE_RUN_STATUS_UNSPECIFIED: "Unknown",
+		datapipelinespb.PipelineRunStatus_PIPELINE_RUN_STATUS_SCHEDULED:   "Scheduled",
+		datapipelinespb.PipelineRunStatus_PIPELINE_RUN_STATUS_STARTED:     "Running",
+		datapipelinespb.PipelineRunStatus_PIPELINE_RUN_STATUS_COMPLETED:   "Success",
+		datapipelinespb.PipelineRunStatus_PIPELINE_RUN_STATUS_FAILED:      "Failed",
+	}
+
+	if len(runs) > 0 {
+		printf(c.App.Writer, "Last run: %s, %s.",
+			runs[0].GetStartTime().AsTime().Format(time.RFC3339),
+			pipelineRunStatusMap[runs[0].GetStatus()])
+	} else {
+		printf(c.App.Writer, "Has not run yet.")
+	}
+
+	return nil
+}
+
+type datapipelineEnableArgs struct {
+	ID string
+}
+
+// DatapipelineEnableAction enables a data pipeline.
+func DatapipelineEnableAction(c *cli.Context, args datapipelineEnableArgs) error {
+	client, err := newViamClient(c)
+	if err != nil {
+		return err
+	}
+
+	_, err = client.datapipelinesClient.EnableDataPipeline(context.Background(), &datapipelinespb.EnableDataPipelineRequest{
+		Id: args.ID,
+	})
+	if err != nil {
+		return fmt.Errorf("error enabling data pipeline: %w", err)
+	}
+
+	printf(c.App.Writer, "data pipeline (id: %s) enabled.", args.ID)
+	return nil
+}
+
+type datapipelineDisableArgs struct {
+	ID string
+}
+
+// DatapipelineDisableAction disables a data pipeline.
+func DatapipelineDisableAction(c *cli.Context, args datapipelineDisableArgs) error {
+	client, err := newViamClient(c)
+	if err != nil {
+		return err
+	}
+
+	_, err = client.datapipelinesClient.DisableDataPipeline(context.Background(), &datapipelinespb.DisableDataPipelineRequest{
+		Id: args.ID,
+	})
+	if err != nil {
+		return fmt.Errorf("error disabling data pipeline: %w", err)
+	}
+
+	printf(c.App.Writer, "data pipeline (id: %s) disabled.", args.ID)
+	return nil
+}
+
+func parseMQL(mql, mqlFile string) ([][]byte, error) {
+	if mqlFile != "" {
+		if mql != "" {
+			return nil, errors.New("data pipeline MQL and MQL file cannot both be provided")
+		}
```

Comment:
Question: And totally optional. Would this read better if it was a separate check?

`if mqlFile != "" && mql != "" {...}`